Integrations/integration-MISP_V2.yml

```diff
@@ -1642,7 +1678,23 @@ script:
       description: All tag names in the event
       type: string
     description: Adds a tag to the given UUID event or attribute.
+  - name: misp-add-sighting
+    arguments:
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - sighting
+      - false-positive
+      - expiration
+      description: Type of sighting to add
+    - name: id
+      description: ID of attribute to add sighting to
```

Comment:
Please add to the description where this `ID` can be retrieved from, like from which command.
And not sure I understand why this or `UUID` are not mandatory

Reply:
one of them is mandatory, not both. If none will be filled exception will be thrown. will add to description

Reply:
What error will be thrown? from the SDK? can you please post the error message?

Reply:
added better error than one raised from the sdk